### ISSUE TITLE
lines-between-class-members/exceptBetweenSingleLine

### ISSUE THREAD

**Issue by LukeSamkharadze** [eslint/eslint#14665]
**What rule do you want to change?**
Lines-between-class-members

**Does this change cause the rule to produce more or fewer warnings?**
More

**How will the change be implemented? (New option, new default behavior, etc.)?**
New option for `lines-between-class-members` that will place newlines **before** & **after** a multiple-line member

**Please provide some example code that this change will affect:**

### Incorrect
```js
/* eslint lines-between-class-members: ["error", "always", { "exceptBetweenSingleLine": true }]*/
class Foo{
  bar(){} // single line class member
  qux(){} // single line class member
  baz(){
    // multi line class member
  }
}
```
### Correct
```js
/* eslint lines-between-class-members: ["error", "always", { "exceptBetweenSingleLine": true }]*/
class Foo{
  bar(){} // single line class member
  qux(){} // single line class member
  
  baz(){
    // multi line class member
  }
}
```

**What does the rule currently do for this code?**
Place new lines **after** any multiple-line member

**What will the rule do after it's changed?**
Place new lines **before** & **after** any multiple-line member

**Are you willing to submit a pull request to implement this change?**
Cant 😞

**Comment by LukeSamkharadze** [comment]
In the case of the current option 

This would be correct

```js
/* eslint lines-between-class-members: ["error", "always", { "exceptAfterSingleLine": true }]*/
class Foo{
  bar(){} // single line class member
  qux(){} // single line class member
  baz(){
    // multi line class member
  }
}
```

**Comment by nzakas** [comment]
Thanks for reaching out. Per our [stylistic rule policy](https://github.com/eslint/eslint#stylistic-rule-updates), we aren’t adding any more options to this rule.
